projects/librdkafka.cmake

```diff
@@ -1,13 +1,13 @@
 ########################################
-# qt5
+# librdkafka
 ########################################
 xpProOption(librdkafka)
-set(KAFKA_VER 0.9.0)
+set(KAFKA_VER master)
 set(KAFKA_REPO https://github.com/edenhill/librdkafka)
```

Comment:
Using master makes it kind of tricky to keep track of where we were at in any particular ovsrpro build - do we want to somehow capture the hash of the last commit or something so that in the built version of ovsrpro we know what version of librdkafka is in it?